docusaurus/docs/learn/quickstarts/network/miniziti.bash

```diff
@@ -774,34 +792,51 @@ main(){
         testClusterDns "${MINIKUBE_NODE_EXTERNAL}"
     fi
 
+    if miniziti kubectl get configmaps \
+        | grep -q "$MINIZITI_INGRESS_ZONE_CONFIGMAP"; then
```

Comment:
`kubectl get cm` will return non-zero if a name is provided to eliminate the quiet grep and eliminate the risk of an unintentional substring match from some other cm name.

```
kubectl get configmaps $MINIZITI_INGRESS_ZONE_CONFIGMAP
```

Reply:
Will update.

Reply:
This should be resolved.